platform/src/main/java/org/stellar/anchor/platform/config/PropertyAppConfig.java

```diff
@@ -2,21 +2,25 @@
 
 import java.util.List;
 import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.validation.Errors;
 import org.springframework.validation.ValidationUtils;
 import org.springframework.validation.Validator;
 import org.stellar.anchor.config.AppConfig;
-import org.stellar.anchor.platform.service.SpringResourceReader;
-import org.stellar.anchor.util.ResourceReader;
 import org.stellar.anchor.util.UrlValidationUtil;
 
 @Data
 public class PropertyAppConfig implements AppConfig, Validator {
-  private String stellarNetworkPassphrase = "Test SDF Network ; September 2015";
-  private String hostUrl = "http://localhost:9800";
-  private String horizonUrl = "https://horizon-testnet.stellar.org";
-  private String jwtSecretKey;
-  private String assets = "assets-test.json";
+  @Value("${stellar_network.network_passphrase}")
+  private String stellarNetworkPassphrase;
+
+  @Value("${host_url}")
+  private String hostUrl;
```

Comment:
Is this annotation needed? I thought it could be inferred.

Reply:
It may not be necessary because of Spring's relaxed binding. But they are made to be explicit so that `hostUrl` will not be acceptable.